platform/cypress/e2e/collocation/collocation.cy.js

```diff
@@ -0,0 +1,157 @@
+describe('Collocation feature', () => {
+  it('schedule devices for collocation', () => {
+    // Intercept the API request to get the device status summary
+    cy.intercept('GET', '**/events/running?tenant=airqo', {
+      statusCode: 200,
```

Comment:
Glad that we are doing unit testing! Thanks @Codebmk

Reply:
Yeah @Baalmart I am able to build the application on my machine but it failed here. cc: @123MwanjeMike 
![Screenshot from 2023-03-28 22-23-47](https://user-images.githubusercontent.com/46527380/228442818-38bca604-4340-4df5-87c1-b3271cec3589.png)

Reply:
Great work @Codebmk for both the unit and end to end(e2e) tests. Thanks.
However, the test (not deploy preview job @Baalmart ) is failing because of the TypeError below.
Could you look into the newly created functions to ensure they all have values for their variables or something like that @Codebmk 
![image](https://user-images.githubusercontent.com/41787587/228499455-b51fd4ca-374e-4b90-9282-32a494ac3eae.png)